swhkd/src/daemon.rs

```diff
@@ -197,11 +215,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
         (Key::KEY_RIGHTSHIFT, config::Modifier::Shift),
     ]);
 
-    let repeat_cooldown_duration: u64 = if args.is_present("cooldown") {
-        args.value_of("cooldown").unwrap().parse::<u64>().unwrap()
-    } else {
-        250
-    };
+    let repeat_cooldown_duration: u64 = args.cooldown.unwrap_or(250);
```

Comment:
This 250 should not have been hardcoded, seems like this was an oversight in the old commits too. Can this be extracted to a constant?

Reply:
Apart from this, the rest looks fine to me from a preliminary view.

Reply:
How do you want it to be? Do you want me to define a separate constant somewhere and use it in `unwrap_or`? Or are you asking for something else?

Reply:
Apologies for the late response, a constant will do.

Reply:
Sorry for pushing the changes so late. A fix has been committed